src/librbd/operation/DisableFeaturesRequest.cc

```diff
@@ -178,7 +178,9 @@ Context *DisableFeaturesRequest<I>::handle_acquire_exclusive_lock(int *result) {
     if ((m_features & RBD_FEATURE_EXCLUSIVE_LOCK) != 0) {
       if ((m_new_features & RBD_FEATURE_OBJECT_MAP) != 0 ||
           (m_new_features & RBD_FEATURE_JOURNALING) != 0) {
-        lderr(cct) << "cannot disable exclusive lock" << dendl;
+        lderr(cct) << "cannot disable exclusive-lock feature. feature "
+                      "object-map or journaling must be disabled "
+                      "before disabling exclusive-lock." << dendl;
```

Comment:
minor: better to keep it as "cannot disable exclusive-lock. object-map and journaling ..." rather than using "feature" word everytime?

applies to changes elsewhere.